Library/Homebrew/cask/utils.rb

```diff
@@ -7,6 +7,18 @@
 
 module Cask
   module Utils
+    def self.only_gain_permissions(path, command: SystemCommand)
+      if path.respond_to?(:rmtree) && path.exist?
```

Comment:
When would it not respond to `rmtree`?

Reply:
Judging from the caller, it always should. I'll rewrite this with your comment below.